internal/cli/main.go

```diff
@@ -227,15 +227,25 @@ Database drivers: `+strings.Join(database.List(), ", ")+"\n")
 		}
 
 	case "drop":
-		log.Println("Are you sure you want to drop the entire database schema? [y/N]")
-		var response string
-		fmt.Scanln(&response)
-		response = strings.ToLower(strings.TrimSpace(response))
-
-		if response == "y" {
-			log.Println("Dropping the entire database schema")
-		} else {
-			log.fatal("Aborted dropping the entire database schema")
+		dropFlagSet := flag.NewFlagSet("drop", flag.ExitOnError)
+		responseYes := dropFlagSet.Bool("y", false, "Drop the entire database schema")
```

Comment:
Not sure about using `-y`. Most other CLIs use `-f` for this behavior.

The flag help text should be more descriptive.
If we use `-f`, something like `Force the drop command by bypassing the confirmation prompt` might work.
Any ideas for a short descriptive help text?

Reply:
Thanks for your confirmation!

You are certainly right.
I'm sure the help text you wrote is fine too.

I’ve added commits for fix. Could you check it? :pray: